### PR TITLE
Develop Branch - Pull Request

### PR DESCRIPTION
**Pull request by mbaptiste**
Drought monitor scripts and configuration files.

### CODE REVIEW

File: monitor/io.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    for line in old_file:
+        new_file.write(line.replace(pattern, subst))
+    #close temp file
+    new_file.close()
+    os.close(fh)
+    old_file.close()
+    #Remove original file
+    os.remove(file_path)
+    #Move new file
+    move(abs_path, file_path)
```
Comment (jhamman) [inline monitor/io.py:40]: I think @lizaclark's version of this function is better: https://github.com/UW-Hydro/overtheloop/blob/develop/tools/bin/ecflow/core/model_tools.py#L26-L39

Comment (jhamman) [inline monitor/io.py:40]: hmmm., I see you have that in `monitor/io.py` but you don't seem to be using it.

**Comment by mbaptiste** [comment]
Closed and reopened against develop

File: monitor/io.py (hunk 1 above)
Comment (lizaclark) [inline monitor/io.py:40]: She uses it in `tools/bin/ecflow/processes/main/meteorological/get_metdata.py`

**Pull request closed without merging**